releasenotes/notes/accept_level_1_options-06c09a67a9c23feb.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
```

Comment:
Personally I'd list this under `features` instead of `upgrade`, but it's not a big deal.